cpp/src/value_classes/ValueButton.cpp

```diff
@@ -96,6 +96,17 @@ bool ValueButton::PressButton
 }
 
 //-----------------------------------------------------------------------------
+// <ValueButton::ResetButton>
```

Comment:
hrm. This should also generate a ValueChanged Notification so a application can update its state.